mailbox/backup/src/main/java/org/apache/james/mailbox/backup/WithZipHeader.java

```diff
@@ -0,0 +1,35 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox.backup;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import org.apache.james.mime4j.Charsets;
+
+public interface WithZipHeader {
+
+    static int toLittleEndian(String s) {
+        if (s.length() == 2) {
+            return ByteBuffer.wrap(s.getBytes(Charsets.US_ASCII)).order(ByteOrder.LITTLE_ENDIAN).getShort();
+        } else {
```

Comment:
We have different habit in the team.

Personnally, I don't like `else` when you are returning/throwing in the `if` clause.
But I'm maybe the only one.